src/main/resources/db/migration/V1_66__create_missing_supplier_assessments.sql

```diff
@@ -0,0 +1,7 @@
+create extension pgcrypto;
+
+insert into supplier_assessment (id, referral_id)
+    select gen_random_uuid(), r.id
```

Comment:
[uuid-ossp](https://www.postgresql.org/docs/10/uuid-ossp.html) has smaller scope than the [pgcrypto](https://www.postgresql.org/docs/10/pgcrypto.html) extension, so might be better:

```suggestion
create extension if not exists "uuid-ossp";

insert into supplier_assessment (id, referral_id)
    select uuid_generate_v4(), r.id
```

Reply:
ignore me, the docs advise to use pgcrypto :)

> If you only need randomly-generated (version 4) UUIDs, consider using the gen_random_uuid() function from the pgcrypto module instead.